sources/packages/web/src/views/institution/Login.vue

```diff
@@ -53,17 +58,23 @@ export default {
       required: false,
       default: false,
     },
+    showUnknownUserMessage: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
   },
   setup(props: any) {
     const basicBCeID = ref(props.showBasicBCeIDMessage);
     const disabledUser = ref(props.showDisabledUserMessage);
+    const unknownUser = ref(props.showUnknownUserMessage);

```

Comment:
I do not see any reason to use "ref" here since we can directly access the properties on the template.
Since the others are already following this it will not be a problem for this PR.
In the future, we do not need to use ref either return them on the setup method.